docs/content/en/docs/quickstart/_index.md

```diff
@@ -26,15 +26,18 @@ cd manifests
 ### 2. Installing control plane
 
 ``` console
-helm install pipecd ./manifests/pipecd --values ./quickstart/control-plane-values.yaml
+kubectl create namespace pipecd
+helm install pipecd ./manifests/pipecd \
+  --values ./quickstart/control-plane-values.yaml \
+  --namespace pipecd
```

Comment:
nit: Let's move the `namespace` option to be the first one in order.

```
helm install pipecd ./manifests/pipecd \
  --namespace pipecd \
  --values ./quickstart/control-plane-values.yaml
```

Reply:
Right, it's more conventional.